app/scripts/modules/core/src/help/help.contents.ts

```diff
@@ -103,6 +103,12 @@ module(HELP_CONTENTS, [])
         <p>The GCS object name, in the form 'gs://bucket/path/to/file.yml.'</p>`,
     'pipeline.config.expectedArtifact.docker.name': `
         <p>The Docker image name you want to trigger on changes to. If you are treating a tag as a release stream, you should include the image tag in the name. An example is 'gcr.io/project/image:stable', where all changes to the 'stable' tag will result in triggering this pipeline.</p>`,
+    'pipeline.config.trigger.webhook.source': `
+        <p>Determines the target URL required to trigger this pipeline, as well as how the payload can be transformed into artifacts.</p>
+    `,
+    'pipeline.config.trigger.webhook.constraints': `
+        <p>When provided, only a webhook containing at least the specified key/value pairs will be allowed to trigger this pipeline. For example, if you wanted to lockdown the systems/users that can trigger this pipeline via this webhook, you could require the key "secret" and value "something-secret" as a constraint</p>
```

Comment:
Missing a period at the end.

Reply:
Key/value pairs in the query parameters? In the request body?

Reply:
In the payload, will address